apiary.apib

```diff
@@ -4013,6 +3931,55 @@ Update of the credentials for BigQuery backend.
 
 Create a new Snowflake storage backend with RSA certificate authentication. The backend is created as disabled and returns an SQL template for setting up Snowflake roles and users. After running the generated query, you can activate the backend using the [Activate Snowflake Storage Backend](#reference/super-storage-backends-management/activate-snowflake-storage-backend/activate-snowflake-backend) endpoint.
 
+Registration will provide personalized queries based on this template
+```
+// base
+CREATE ROLE "KBC_STORAGE";
```

Comment:
zkopiroval jsem to z unit testu. abychom videli co po klientech chceme v dokumentaci a nemuseli to nekde tahat z kodu